packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.Props.ts

```diff
@@ -43,4 +47,9 @@ export interface IBreadcrumbItem {
    * Url to navigate to when this breadcrumb is clicked.
    */
   href?: string;
+
+  /**
+   * Optional arbitrary user-defined data
+   */
+  data?: any;
```

Comment:
not sure about this, shouldn't the key be enough for a user to get any meta-data that might be required?